### PR TITLE
Issue #4538: Update documentation in user.actions.inc.

### LINKED ISSUE #4538: [DX] Wrong description for user_add_role_action()
https://api.backdropcms.org/api/backdrop/1/search/user_add_role_action

The description says:

User $account: The user entity whose account will be blocked.

### PR DESCRIPTION
**Pull request by ghost**
Fixes https://github.com/backdrop/backdrop-issues/issues/4538

### CODE REVIEW

File: core/modules/user/user.actions.inc [hunk 1]
```diff
@@ -77,7 +77,7 @@ function user_cancel_user_action($account, &$context) {
  * Adds a role to a user account.
  *
  * @param User $account
- *   The user entity whose account will be blocked.
+ *   The user entity whose account will be updated.
```
Comment (klonos) [inline core/modules/user/user.actions.inc]:
```suggestion
 *   The user entity to which the role will be added.
```

File: core/modules/user/user.actions.inc [hunk 2]
```diff
@@ -93,7 +93,7 @@ function user_add_role_action($account, &$context) {
  * Removes a role from a user account.
  *
  * @param User $account
- *   The user entity whose account will be blocked.
+ *   The user entity whose account will be updated.
```
Comment (klonos) [inline core/modules/user/user.actions.inc]:
```suggestion
 *   The user entity from which the role will be removed.
```

**Review by klonos — APPROVED** [review]
LGTM 👍

**Comment by ghost** [comment]
Thanks, merged into 1.x and 1.16.x.

**Pull request merged**